go/host/node/node.go

```diff
@@ -562,6 +601,20 @@ func (a *Node) storeBlockProcessingResult(result common.BlockSubmissionResponse)
 	}
 }
 
+// Distributes logs to subscribed clients.
+// TODO - #453 - Encrypt logs, rather than just serialising them as JSON.
+// TODO - #453 - Distribute logs specifically based on subscription IDs, rather than sending all logs to everyone.
+func (a *Node) sendLogsToSubscribers(result common.BlockSubmissionResponse) {
+	for _, jsonLogs := range result.SubscribedLogs {
```

Comment:
I see there are todos here which I think capture the requirements.
Basically, based on the `result.SubscribedLogs` key, this has to do the dispatching of encrypted events. There should be no unmarshalling/unencryption here.